packages/localize-tools/src/messages.ts

```diff
@@ -66,6 +66,7 @@ export interface Bundle {
  */
 export interface Placeholder {
   untranslatable: string;
+  index?: string;
```

Comment:
Number instead of string here I think?

Reply:
Here and for the comment below, I chose string since when these are added as attributes on the placeholder tags in the translation, they become strings and I'm eventually looking up by that. I guess I could cast that to a number when comparing too, so it could be either.